doc/developer/scan.txt

```diff
@@ -132,22 +63,266 @@ files, and section of the code they deal with, are :
 * ``scan_views.py`` contains different views of the scan op that have
   simpler and easier signatures to be used in specific cases.
 
-* ``scan_opt.py`` contains the list of all optimizations for the scan
-  operator.
+* ``scan_opt.py`` contains the list of all Theano graph optimizations for the
+  scan operator.
+
+
+Notation
+========
+
+Scan being a sizeable and complex module, it has its own naming convention for
+functions and variables which this section will attempt to introduce.
+
+Recall that a Scan op contains a Theano function representing the computation
```

Comment:
I think you should remove the "Recall that" there is nothing to recall as it was never mentioned in this document before.